### DISCUSSION TITLE
Is there any way to add tilling ?

### DISCUSSION THREAD

**Discussion by atul161** [google/filament#6497] [Q&A] [answered]
Hi team!

I am trying to add tilling support. I have a cube glb and I have another model which is inside that cube. Just want to know that is there any way to add tilling. I have attached the texture as an image there but the image gets stretched. I want to repeat the image.

I don't found a proper way to achieve this.

**Comment by bejado** [comment, marked as answer]
One way to tile the texture is by adjusting the UV coordinates of your model to extend beyond 0 to 1 and setting the `TextureSampler`'s `WrapMode` to `REPEAT`.

**Reply by atul161** [reply]
Thanks @bejado !
